index.js

```diff
@@ -281,7 +281,17 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      if (Object.prototype.toString.call(key) === '[object Array]') {
```

Comment:
If that's the case, we should really refactor all the `var` declarations to `const` and `let`, too.